beetmoverscript/__init__.py

```diff
@@ -1,3 +0,0 @@
-from ._version import __version__
-
-__all__ = ['__version__']
```

Comment:
`__version__` is used by towncrier, but it can be given to it explicitly by specifying `--version`. Can you update README.md accordingly

Reply:
done